File: 1_iniciante/javascript/1016.js
/**
 * @source Beecrowd
 * @problem 1016 - Distance
 * @subject Sequential
 *
 * @author Letícia Cruz
 * @date 2020-10-02
 */

var input = require("fs").readFileSync("/dev/stdin", "utf8");
var lines = input.split("\n");

let minutos = (60 * input) / 30;

console.log(minutos + " minutos");

File: 1_iniciante/java/1016.java
/**
 * @source Beecrowd
 * @problem 1016 - Distance
 * @subject Sequential
 * 
 * @author Letícia Cruz
 * @date 2025-07-06
 */

import java.io.IOException;
import java.util.Scanner;

class Main {

    public static void main(String[] args) throws IOException {

        // letirura de entradas
        Scanner scanner = new Scanner(System.in);
        int kms = scanner.nextInt();

        /*
         * tempo que carro y leva para passar n kms do carro x: 1 km a cada 2 minutos
         * 
         * obs: a grandeza já fica em minutos, logo, não precisa de conversão
         * 
         */

        int tempoEmMinutos = kms * 2;
        System.out.println(tempoEmMinutos + " minutos"); // formado de saída específicada

        scanner.close();
    }
}
